addons/website/static/tests/tours/edit_link_popover.js

```diff
@@ -96,11 +123,16 @@ tour.register('edit_link_popover', {
     },
     {
         content: "Click on the Home menu again",
-        trigger: '#top_menu a:contains("Home")',
         extra_trigger: '#top_menu a:contains("Home")[href="/contactus"]', // href should be changed
+        trigger: '#top_menu a:contains("Home")',
+        // FIXME this run shouldnt be needed but click not working as real click
+        run: (actions) => {
+            actions.click();
+            actions.tip_widget.$anchor.popover('show');
```

Comment:
Note: if I am not mistaken you could achor the anchor via `this.$anchor` (but then you have to define the run function with the keyword `function`

Reply:
Indeed, much cleaner :+1: 
What a memory you have ;-)

Reply:
Well... I created that `this.$anchor` stuff and I would not call it clean ^^ But "cleaner" sure :')